Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Build stage
+FROM node:22-slim AS builder
+
+# Set working directory
+WORKDIR /app
+
+# Copy package files
+COPY package*.json ./
+
+# Install dependencies
+RUN npm ci
+
+# Copy source code
+COPY . .
```

Comment:
nitpick: I would copy only `src` + `tsconfig*`, doesn't matter for CI, but in case somebody has some large files in working dir etc.